packages/hub/src/components/layout/RootLayout/PageFooter.tsx

```diff
@@ -0,0 +1,26 @@
+"use client";
+
+import { isModelRoute } from "@/routes";
+import { usePathname } from "next/navigation";
+import { FC } from "react";
+
+export const PageFooter: FC = () => {
+  const pathname = usePathname();
+  if (isModelRoute(pathname)) {
```

Comment:
This feels a bit hacky, but I'm fine with it.

Reply:
There's no other way to do this that I can think of; page.tsx can't pass this info upstream to the top-level layout.tsx.

Reply:
I imagine we could have page specify a different layout - but this would be a pain.

Reply:
Yep, that'd be awkward with Next.js nested layouts.

There are [route groups](https://nextjs.org/docs/app/building-your-application/routing/route-groups), but then we'd have to provide the entire page layout, with top menu, and the performance would be slightly worse, since usually Next.js doesn't re-render layouts on navigation whenever possible.